### PR TITLE
Remove duplicate pause reason definitions

### PR DESCRIPTION
**Pull request by C-gyorfi**
## Context
Following the conversation on this [PR #388](https://github.com/LBHackney-IT/LBH-IncomeCollection/pull/388/files/922a664cd8cdc698f3101b25e48f0c3d23ee17ad#)
## Changes in this pull request
Remove duplicate pause reason definitions

### CODE REVIEW

File: app/helpers/tenancy_helper.rb [hunk 1]
```diff
@@ -25,6 +25,12 @@ def pause_reasons
     }
   end
 
+  def pause_reason_filter(option)
```
Comment (antonyoneill) [inline app/helpers/tenancy_helper.rb:28]: How is this being referenced?

File: app/helpers/pause_reason_filter_helper.rb [hunk 2]
```diff
@@ -1,16 +0,0 @@
-module PauseReasonFilterHelper
```
Comment (antonyoneill) [inline app/helpers/pause_reason_filter_helper.rb:1]: Do we need to replace the import of this helper with the tenancy helper? or does rails auto import this?

File: app/helpers/tenancy_helper.rb (hunk 1 above)
Comment (C-gyorfi) [reply app/helpers/tenancy_helper.rb:28]: its referenced in the `_pause_reason_filter.html.erb` 
```RUBY
          <%= select_tag(:pause_reason, pause_reason_filter(@filter_params.pause_reason), { :class => 'form-control', prompt: 'Select a pause reason'  }) %>
```
 tenancy_helper.rb  is loaded trough rails_helper.rb as any other helpers, apparently its accecible in any erb file

Comment (antonyoneill) [reply app/helpers/tenancy_helper.rb:28]: Great! Thanks - that's why we don't need to remove any references to the PauseReasonFilterHelper

**Review by antonyoneill — APPROVED** [review]

**Pull request merged**